src/reducers/books.js

```diff
@@ -0,0 +1,3 @@
+const booksReducer = initialState => (state = initialState) => state;
+
+export default booksReducer;
```

Comment:
Your `book.js` is missing the actions needed.
![image](https://user-images.githubusercontent.com/31897434/79816233-a1adeb80-837a-11ea-8115-48b3416c453f.png)
Kindly add these. 👆